application/src/main/java/org/togetherjava/tjbot/commands/CommandHandler.java

```diff
@@ -89,8 +117,10 @@ private void getArgumentsAndCommandByEvent(GenericComponentInteractionCreateEven
 
         if (command != null) {
             List<String> args = new ArrayList<>(Arrays.asList(argsArray));
-            args = args.subList(2, args.size());
-            onSucceed.accept(command, args);
+
+            final List<String> argsExcludingFirst = fullList.subList(2, fullList.size());
+
+            executorService.submit(() -> onSucceed.accept(command, listExcludingStart));
```

Comment:
Apologies, editing code is comments is not ideal ^^

Try this:
```suggestion
            final List<String> argsExcludingFirst = args.subList(2, args.size());
            executorService.submit(() -> onSucceed.accept(command, argsExcludingFirst));
```